### PR TITLE
Use ansible as verifier

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes https://github.com/onaio/ansible-onadata/issues/26

### CODE REVIEW

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  hosts: all
+  tasks:
+    - name: Verify that host file exists.
+      stat:
+        path: "/etc/hosts"
+      register: host_file
+
+    - name: Fail if host file does not exist
+      fail:
+      when: not host_file.stat.exists
```
Comment (DavisRayM) [inline molecule/default/verify.yml:12]: Mhm not entirely sure about the naming of these two tasks... Feels like the first task name is more suitable for the second task here...

**Review by DavisRayM — COMMENTED** [review]
Looks good so far, could we also add tests that ensure the services(`onadata` & celery services if enabled) are up after the script is run?

File: molecule/default/verify.yml (hunk 1 above)
Comment (FrankApiyo) [reply molecule/default/verify.yml:12]: renamed!

**Comment by FrankApiyo** [comment]
> Looks good so far, could we also add tests that ensure the services(`onadata` & celery services if enabled) are up after the script is run?

We are currently not checking if the services are enabled and only if they exist. I am not certain why they are not enabled after `molecule converge` is run. I suggest we look into this issue in a separate PR

**Review by DavisRayM — APPROVED** [review]

**Review by jasonrogena — APPROVED** [review]

**Review by ivermac — APPROVED** [review]

**Review by DavisRayM — APPROVED** [review]

**Pull request merged**